templates/types/multi-agents/fastapi/env.example

```diff
@@ -0,0 +1,37 @@
+# The Llama Cloud API key.
+# LLAMA_CLOUD_API_KEY=
+
+# The provider for the AI models to use.
+MODEL_PROVIDER=openai
+
+# The name of LLM model to use.
+MODEL=gpt-4-turbo
+
+# Name of the embedding model to use.
+EMBEDDING_MODEL=text-embedding-3-large
+
+# Dimension of the embedding model to use.
+EMBEDDING_DIM=1024
+
+# The OpenAI API key to use.
+# OPENAI_API_KEY=
+
+# Temperature for sampling from the model.
+# LLM_TEMPERATURE=
+
+# Maximum number of tokens to generate.
+# LLM_MAX_TOKENS=
+
+# The number of similar embeddings to return when retrieving documents.
+TOP_K=3
+
+# Specify the port for the multi-agent services
+# Core services
+MESSAGE_QUEUE_PORT=8000
+CONTROL_PLANE_PORT=8001
+HUMAN_CONSUMER_PORT=8002
+# Agent services
+AGENT_QUERY_ENGINE_PORT=8003
+AGENT_QUERY_ENGINE_DESCRIPTION="Can query the knowledge about US letter standards"
+AGENT_DUMMY_PORT=8004
+AGENT_DUMMY_DESCRIPTION="Does nothing"
```

Comment:
how about the following:

1. use defaults for the ports in code (then configuration via env variables is optional)
2. inline `AGENT_DUMMY_DESCRIPTION` (no need for an env variable)
3. rename `AGENT_QUERY_ENGINE_DESCRIPTION` to `QUERY_ENGINE_DESCRIPTION`, generate a default in env-variable.ts and also use it for the single-agent's `QueryEngineTool` (Python and TS)

From my understanding, with these changes, we could use the generated .env file

Reply:
Agree with 1. and 2.
About 3. I think we can keep the `AGENT_QUERY_ENGINE_DESCRIPTION`, the reason is the agent description is a bit different with the tool or the index and one agent can use multiple tools/index too. We should work on the tool dynamic description and index description first then we will able to construct the agent from those metadata.